src/cdk/bootstrap.ts

```diff
@@ -212,40 +210,50 @@ export class MiraBootstrap {
         ...process.env
       }
     })
-    await new Promise(resolve => {
-      proc.on('exit', () => {
+    await new Promise((resolve, reject) => {
```

Comment:
Wondering if this `deployCi` fn is where we want to put the try/catch block instead (see above remark)?